test/forking/infura/simple.js

```diff
@@ -0,0 +1,156 @@
+const shouldSkip = require("./skip");
+const Web3 = require("web3");
+var assert = require("assert");
+const Ganache = require(process.env.TEST_BUILD
+  ? "../../../build/ganache.core." + process.env.TEST_BUILD + ".js"
+  : "../../../index.js");
+
+const logger = {
+  log: function(msg) {
+    /* console.log(msg) */
+  }
+};
+
+describe("Simple Infura", function() {
+  let INFURA_KEY;
+
+  before(function() {
+    if (shouldSkip()) {
```

Comment:
Minor nitpick, seems like a premature abstraction to drop this in a new file when its basically a one liner and only used once.

Reply:
I have future plans to use this abstraction later. It also can help others as we add more tests to the infura folder. It will keep this organized so that when someone 6 months from now wants to make a new test needs to figure out how to skip the tests, they don't need to remember the specific environment key

I'm happy to remove it if y'all really want to, but I believe it to be good organization skills

Reply:
I generally subscribe to the rule of three. https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)

Reply:
Im sure its a fine abstraction, its just too few examples to know whether its ultimately the right one.

Reply:
I just know that ganache-core's tests are already a mess and organization is important to prevent that from happening further, but definitely your call. I will remove the abstraction

Reply:
No need to remove it, or at least I would just defer to @davidmurdoch on whether thats actually necessary. As I said, it simply seems premature from my perspective as your intention to use it later on was an unknown to me. I personally wait to abstract until there is actual code duplication as too many abstractions is actually counterproductive to organization. Too many one liner utility functions can become unwieldy.